proof/refine/X64/VSpace_R.thy

```diff
@@ -949,6 +955,8 @@ proof -
     apply (rule corres_fail, clarsimp simp: valid_cap_def)
    apply (simp add: perform_page_invocation_unmap_def performPageInvocationUnmap_def split_def)
     apply (rule corres_guard_imp)
+     apply (rule corres_split'[where r'=dc, OF _ corres_return_eq_same[OF refl]
+                                                  hoare_post_taut hoare_post_taut])
```

Comment:
indentation on these: `hoare_post_taut` is part of the `OF`, not the `corres_return_eq_same` so should be indented in line with the `_`. Same goes for the other copy-pasted instances.